src/routes/user.js

```diff
@@ -137,7 +137,7 @@ router.delete('/:id', async (request, response, next) => {
     next(error);
   }
 });
-
+// why does this route exist when we have the '/' route already doing the same thing? can we delete this
 router.get('/user', async (request, response, next) => {
```

Comment:
Which '/' route are you referring to? All routes are specific to their functions, so if another route is doing this first as a check, then we should not delete this.